### PR TITLE
Fix examples due to removal of deprecated Kafka Streams API

### PR DESCRIPTION
**Pull request by mjsax**

### CODE REVIEW

**Comment by mjsax** [comment]
@miguno -- This PR seems to subsume your change from https://github.com/confluentinc/kafka-streams-examples/issues/112 -- the build passed. Does this resolve the issue?

**Comment by guozhangwang** [comment]
@mjsax thanks for the quick turn around! lgtm.

File: src/main/java/io/confluent/examples/streams/microservices/OrdersService.java [hunk 1]
```diff
@@ -125,11 +130,11 @@ public OrdersService(String host) {
...
    * fulfilled.
    */
-  private KStreamBuilder createOrdersMaterializedView() {
-    KStreamBuilder builder = new KStreamBuilder();
-    builder.stream(ORDERS.keySerde(), ORDERS.valueSerde(), ORDERS.name())
-        .groupByKey(ORDERS.keySerde(), ORDERS.valueSerde())
-        .reduce((agg, newVal) -> newVal, ORDERS_STORE_NAME)
+  private StreamsBuilder createOrdersMaterializedView() {
+    StreamsBuilder builder = new StreamsBuilder();
+    builder.stream(ORDERS.name(), Consumed.with(ORDERS.keySerde(), ORDERS.valueSerde()))
```
[the remark this answers is not in the document]
Comment (mjsax) [reply src/main/java/io/confluent/examples/streams/microservices/OrdersService.java]: Not sure -- I just fixed the syntax -- did not look at the code -- @benstopford should know, as we wrote the code originally.

Fixing it sounds reasonable to me. Thx!

[the remark this answers is not in the document]
Comment (benstopford) [reply src/main/java/io/confluent/examples/streams/microservices/OrdersService.java]: God knows. that was ages ago, but agreed, should be a table. It should be StreamsBuilder too. Both fixes here: https://github.com/confluentinc/kafka-streams-examples/pull/116

**Pull request merged**